modules/hcp-get-ocp-channel.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-updating.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-get-ocp-channel_{context}"]
+= Setting channels in a hosted cluster
+
+You can see available updates in the `HostedCluster.Status` field of the `HostedCluster` custom resource (CR).
+
+The available updates are not fetched from the Cluster Version Operator (CVO) of a hosted cluster. The list of the available updates can be different from the available updates from the following fields of the `HostedCluster` custom resource (CR):
+
+* `status.version.availableUpdates`
+* `status.version.conditionalUpdates`
+
+The initial `HostedCluster` CR does not have any information in the `status.version.availableUpdates` and `status.version.conditionalUpdates` fields. After you set the `spec.channel` field to the stable {product-title} release version, the HyperShift Operator reconciles the `HostedCluster` CR and updates the `status.version` field with the available and conditional updates.
+
+See the following example of the `HostedCluster` CR that contains the channel configuration:
+
+[source,yaml]
+----
+spec:
+  autoscaling: {}
+  channel: stable-4.y <1>
+  clusterID: d6d42268-7dff-4d37-92cf-691bd2d42f41
+  configuration: {}
+  controllerAvailabilityPolicy: SingleReplica
+  dns:
+    baseDomain: dev11.red-chesterfield.com
+    privateZoneID: Z0180092I0DQRKL55LN0
+    publicZoneID: Z00206462VG6ZP0H2QLWK
+----
+<1> Replace `<4.y>` with the {product-title} release version you specified in `spec.release`. For example, if you set the `spec.release` to `ocp-release:4.16.4-multi`, you must set `spec.channel` to `stable-4.16`.
+
+After you configure the channel in the `HostedCluster` CR, to view the output of the `status.version.availableUpdates` and `status.version.conditionalUpdates` fields, run the following command:
+
+[source,terminal]
+----
+$ oc get -n <hosted_cluster_namespace> hostedcluster <hosted_cluster_name> -o yaml
+----
++
```

Comment:
Since thie is outside a step, it doesn't need a `+` (and the `+` makes it render weird). Just a blank line should fix it.
![image](https://github.com/user-attachments/assets/70d52603-3d90-414f-b3e2-0331c04acccc)


```suggestion

```